Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/RequirementNodeData.xml

```diff
@@ -6202,4 +6202,35 @@
     <CRequirementCountUpgrade id="AP_CountUpgradeAP_RoachVileCompleteOnlyTechTreeCheat">
         <Count Link="AP_RoachVile" State="CompleteOnly"/>
     </CRequirementCountUpgrade>
+    <CRequirementCountUpgrade id="AP_CountUpgradeAP_DisruptorPermanentCloakCompleteOnly">
+        <Flags index="TechTreeCheat" value="0"/>
+        <Count Link="AP_DisruptorPermanentCloak" State="CompleteOnly"/>
+    </CRequirementCountUpgrade>
+    <CRequirementCountUpgrade id="AP_CountUpgradeAP_DisruptorRestructuredThrustersCompleteOnly">
+        <Flags index="TechTreeCheat" value="0"/>
+        <Tooltip value="RequirementNode/Tooltip/##id##"/>

```

Comment:
These tooltip values aren't necessary and can be removed; this is already the default. Just make sure the tooltips you add in gamestrings.txt match the updated requirement node IDs.